doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    system.thermostat.turn_off()
</pre><p>Why? People will ask why you are doing things like this. If you had to ask someone for help in making the script, the people in the tutorial will be at the same level but also without the ML knowledge. Explain everything.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>